### DISCUSSION TITLE
Prevent Fall Damage from Horizontal Portals

### DISCUSSION THREAD

**Discussion by Onemario1234** [iPortalTeam/ImmersivePortalsMod#516] [Ideas]
This is an issue I _thought_ was a no brainer. Whoops.

Regardless, there should be an option for portal creation to negate fall damage after falling through a horizontal portal. I recommend this because, using the example datapack, I made a Twilight Forest-like portal to the Skylands, and every time I use the portal, I take damage. Unless I make supports to land on, I can't really prevent this.

**Comment by qouteall** [comment]
It's being considered
